### PR TITLE
Add Payout Method for Draft Expenses As Well

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5359

image

### CODE REVIEW

File: components/expenses/ExpensePayeeDetails.js [hunk 1]
```diff
@@ -186,7 +186,10 @@ const ExpensePayeeDetails = ({ expense, host, isLoading, borderless, isLoadingLo
             />
           </Box>
           <div data-cy="expense-summary-payout-method-data">
-            <PayoutMethodData payoutMethod={expense.payoutMethod} isLoading={isLoadingLoggedInUser} />
+            <PayoutMethodData
+              payoutMethod={expense.draft ? expense.draft?.payoutMethod : expense.payoutMethod}
```
Comment (Betree) [inline components/expenses/ExpensePayeeDetails.js:190]: We can simplify this condition, it's double-checking the existence of `expense.draft`

```suggestion
              payoutMethod={expense.draft?.payoutMethod ?? expense.payoutMethod}
```

**Review by Betree — COMMENTED** [review]
Looks like a good fix from looking at the code. What does it looks like if you submit a draft with no payout method set altogether?

File: components/expenses/ExpensePayeeDetails.js (hunk 1 above)
Comment (SudharakaP) [reply components/expenses/ExpensePayeeDetails.js:190]: Done. 👍🏽

**Comment by SudharakaP** [comment]
> Looks like a good fix from looking at the code. What does it looks like if you submit a draft with no payout method set altogether?

That's handled already. Then we say it's not set;

image

**Review by Betree — APPROVED** [review]
Perfect

**Pull request merged**